tests/cms/views/poi_categories/test_poi_category_form_view.py

```diff
@@ -0,0 +1,500 @@
+            "translations-MIN_NUM_FORMS": "10",
+            "translations-MAX_NUM_FORMS": "10",
+        },
+    )
+
+    edit_url = response.headers.get("location")
+    assert response.status_code == 302
+
+    id_of_poicategory = resolve(edit_url).kwargs["pk"]
+
+    poicategory = POICategory.objects.get(id=id_of_poicategory)
+    translation = poicategory.translations.get(language__slug="de")
+
+    response = client.post(
+        edit_url,
+        data={
+            "icon": "daily_routine",
+            "color": "#1DC6C6",
+            "translations-1-category": "",
+            "translations-1-language": "8",
+            "translations-1-id": "",
+            "translations-1-name": "",
+            "translations-2-category": "",
+            "translations-2-language": "3",
+            "translations-2-id": "",
+            "translations-2-name": "",
+            "translations-0-category": str(poicategory.id),
+            "translations-0-language": "1",
+            "translations-0-id": str(translation.id),
+            "translations-0-name": "Nicht veränderte Ortskategorie",
+            "translations-3-category": "",
+            "translations-3-language": "6",
+            "translations-3-id": "",
+            "translations-3-name": "",
+            "translations-4-category": "",
+            "translations-4-language": "10",
+            "translations-4-id": "",
+            "translations-4-name": "",
+            "translations-5-category": "",
+            "translations-5-language": "2",
+            "translations-5-id": "",
+            "translations-5-name": "",
+            "translations-6-category": "",
+            "translations-6-language": "5",
+            "translations-6-id": "",
+            "translations-6-name": "",
+            "translations-7-category": "",
+            "translations-7-language": "4",
+            "translations-7-id": "",
+            "translations-7-name": "",
+            "translations-8-category": "",
+            "translations-8-language": "7",
+            "translations-8-id": "",
+            "translations-8-name": "",
+            "translations-9-category": "",
+            "translations-9-language": "9",
+            "translations-9-id": "",
+            "translations-9-name": "",
+            "translations-TOTAL_FORMS": "10",
+            "translations-INITIAL_FORMS": "1",
+            "translations-MIN_NUM_FORMS": "10",
+            "translations-MAX_NUM_FORMS": "10",
+        },
+    )
+
+    assert response.status_code == 302
+    response = client.get(edit_url)
+    assert response.status_code == 200
+
+    assert "Keine Änderungen vorgenommen" in response.content.decode("utf-8")
+
+
+@pytest.mark.parametrize(
+    "login_role_user", [CMS_TEAM, ROOT, SERVICE_TEAM], indirect=True
+)
+@pytest.mark.django_db
+def test_delete_unused_poi_category_was_successful(
+    load_test_data: None,
+    login_role_user: tuple[Client, str],
+) -> None:
+    UNUSED_POI_CATEGORY_ID = 3
```

Comment:
Is there a specific reason that you rely on a poi-category in the test data here? I am in general not against it, but you tried to avoid relying on existing poi categories in the tests above.

Reply:
I changed it all to post requests now :) Please check if it is as you had in mind :)